src/lib/utils/application-commands/getNeededParameters.ts

```diff
@@ -28,6 +28,7 @@ async function fetchGuildCommands(commands: ApplicationCommandManager) {
 			if (preventFailedToFetchLogForGuilds === true) continue;
 
 			if (Array.isArray(preventFailedToFetchLogForGuilds) && !preventFailedToFetchLogForGuilds?.includes(guildId)) {
+				const guild = container.client.guilds.resolve(guildId)!;
```

Comment:
Shouldn't there be a check in case `guild` is `null`?

Reply:
oh hrm trueeeeeeeeeee